docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,24 +105,52 @@ To test if the Trigger CR is properly connected to the function:
     }
     ```
 
-2. Send an event manually to trigger the function:
+2.  Send an event manually to trigger the function. In the first example, the payload complies with the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/spec.md) specification and the event is sent directly to Eventing Mesh. If you want to send the events to compatibility layer which forwards them to Eventing Mesh, use the second example. 
+
+   <div tabs name="examples" group="test=trigger">
+    <details>
+    <summary label="CloudEvents">
+    CloudEvents
+    </summary>
 
     ```bash
-    curl -X POST -H "Content-Type: application/json" https://gateway.{CLUSTER_DOMAIN}/$APP_NAME/v1/events -k --cert {CERT_FILE_NAME}.crt --key {KEY_FILE_NAME}.key -d \
-    '{
-        "event-type": "{EVENT_TYPE}",
-        "event-type-version": "{EVENT_VERSION}",
-        "event-time": "2020-04-02T21:37:00Z",
-        "data": "123456789"
-    }'
+    curl -v -k --cert {CERT_FILE_NAME} --key {KEY_FILE_NAME} -d '{
```

Comment:
Seems we have to set the datacontenttype (when sending via structured mode, in binary mode it is inferred by the content-type as far as I know), otherwise the data is empty (`{}`) in the lambda.
See suggestion